.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# CMakeLists.txt whitespace fixups
+3e14f865d30271c74fc68d417af488ea91b66d48
```

Comment:
The important parts look good to me, but I'm not sure it's worth having this file at the top level. I might be biased though, becuase I use fugitive.vim, which makes it easy to step across bulk commits with `~`. What do you think?

Reply:
The GitHub Blame view respects this file, i.e. this is useful for anyone who just takes a glance at the code through the Web UI. Also VSCode has an inline line blame which becomes less useful if style commits aren't ignored.

And while it's usually easy to step through the latest few commits, it gets especially tedious after style changing commits layer up deep in the history.